src/server_manager/install_scripts/install_server.sh

```diff
@@ -131,12 +149,11 @@ function verify_docker_running() {
 }
 
 function install_docker() {
-  curl -sS https://get.docker.com/ | sh > /dev/null 2>&1
+  curl -sS https://get.docker.com/ | sh >&2
 }
 
 function start_docker() {
-  systemctl start docker.service > /dev/null 2>&1
-  systemctl enable docker.service > /dev/null 2>&1
+  systemctl enable --now docker.service >&2
```

Comment:
Why are you removing the start command? Is enable guaranteed to call start?

Reply:
`enable` doesn't call `start`, but `enable --now` does.  The manpage says

> enable UNIT..., enable PATH...
> ... Note that this does not have the effect of also starting any of the units being enabled. If this is desired, combine this command with the --now switch, or invoke start with appropriate arguments later.